.pipelines/pr-e2e.yaml

```diff
@@ -52,6 +52,9 @@ jobs:
   - script: make validate-copyright-headers test-style
     displayName: Run linting rules
     workingDirectory: $(modulePath)
+  - script: make validate-generated
+    displayName: Check if generated code is up to date
```

Comment:
```suggestion
    displayName: Check if the generated code is in sync with the static assets.
```

Reply:
I prefer the original wording, especially in context at VSTS. @Azure/aks-engine opinions?

![screen shot 2019-02-08 at 12 41 56 pm](https://user-images.githubusercontent.com/73019/52502316-b9e38180-2b9f-11e9-84d5-34d35e7fc9dc.png)

Reply:
Sure!